permissions/plugin-aws-sqs.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "aws-sqs-plugin"
+paths:
+- "org/jenkins-ci/plugins/aws-sqs-plugin"
```

Comment:
Neither the `groupId` nor the `artifactId` part match what's specified in https://github.com/jenkinsci/aws-sqs-plugin/blob/master/pom.xml